packages/terra-layout/src/HoverTarget.scss

```diff
@@ -0,0 +1,85 @@
+:local {
+  .hover-target {
+    overflow: hidden;
+    width: 100%;
+  }
+
+  .content-section {
+    float: left;
+    -webkit-overflow-scrolling: touch;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+    overflow-y: auto;
+    width: calc(100% - var(--layout-menu-hover-target-width, 30px));
```

Comment:
Does the calc value work as expected in IE10? I am wondering if we need to add 
`width: calc(100% - 30px);` above

Reply:
The fallback will get generated automatically, won't it?

Reply:
Would be good to test, but I believe this compiles to `width: calc(100% - 30px);` in IE.

Reply:
Can update the fallback to `2.143rem`